server/pbenchinacan/deploy-dependencies

```diff
@@ -42,6 +42,8 @@ PB_KEYCLOAK_IMAGE_PULL_POLICY=${PB_KEYCLOAK_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE
 # The overall pod name is derived from the pbench server image tag.
 PB_POD_NAME=${PB_POD_NAME:-"pbench-in-a-can_${PB_SERVER_IMAGE_TAG}"}
 
+PB_DEPLOY_FILES=${PB_DEPLOY_FILES:-"/var/tmp/pbench/pbench_server_deployment"}
```

Comment:
> Are you aware of a case where this file is run separately?

When I want to only re-deploy just dependencies :smile: but mostly debugging purpose. In the past when I did some experiments with our dependency containers I would just redeploy the dependencies.

Reply:
I guess, since its already exported in `run-pbench-in-a-can` script I think I can just get rid of defining it again here.

Reply:
> In the past when I did some experiments with our dependency containers I would just redeploy the dependencies

That's fair.  So, if you want to continue to support that, without requiring the user to define `PB_DEPLOY_FILES` manually, then you need a change like this one...but it really should use the default which the outer layer would have used (because I'm not hearing any reason here why it should have it's own default value which is different).

Reply:
Yes in general I think its best to modify the outer layer (in this case `run-pbench-in-a-can`) so for my example I could just comment out the `server/pbenchinacan/deploy` line and it will only deploy dependencies and vice-versa.